### ISSUE TITLE
IsCluster ambiguity in projectes using both Microsoft.EntityFrameworkCore.SqlServer and EntityFrameworkCore.Jet.Odbc

### ISSUE THREAD

**Issue by angelofb** [bubibubi/EntityFrameworkCore.Jet#140]
The call is ambiguous between the following methods or properties: 'Microsoft.EntityFrameworkCore.JetKeyBuilderExtensions.IsClustered(Microsoft.EntityFrameworkCore.Metadata.Builders.KeyBuilder, bool)' and 'Microsoft.EntityFrameworkCore.SqlServerKeyBuilderExtensions.IsClustered(Microsoft.EntityFrameworkCore.Metadata.Builders.KeyBuilder, bool)'

![immagine](https://github.com/bubibubi/EntityFrameworkCore.Jet/assets/2032257/1eb9d29a-4c56-4b43-9793-f7dc17bd34df)

I have a single project where I put all db logic for various database providers. If I add EntityFrameworkCore.Jet.Odbc the DbContext scaffolded from MsSql rises an error because of the ambiguity of IsClustered method

**Comment by ChrisJollyAU** [comment]
The IsClustered Fluent API was originally added as a compatibility no-op so that the Fluent API usage didn't have to be changed when switching between Sql Server and Jet. Although it seems it has added further problems.

The problem is we are adding an extension method onto the KeyBuilder class and both Sql Server and Jet are defining it with the same name and parameters.

Unfortunately whether you use UseJet or UseSqlServer you get the same KeyBuilder object and the static extension methods can't figure out which one to use.

As previously mentioned the IsClustered does not really do anything in Jet I have removed it.

On a similar note I picked up some other problems of the same type with the ModelBuilder and PropertyBuilder. Looks to be just with UseIdentityColumn(s) at the moment which has been changed to UseJetIdentityColumn(s)

Fixed with fd6c2aec01b2f8554dcc4929c1ce353c226436a8 and fd57f4a3bd3111616fb4f63c368e4d5f7d527967 

Published in 7.0.2

You will likely need to do some if statements to check which provider and use the correct fluent api for that provider
